persia/ctx.py

```diff
@@ -192,6 +307,10 @@ def __enter__(self):
         )
         self.sparse_optimizer.apply()
 
+        if self.initial_embedding_dir is not None:
+            self.load_embedding(self.initial_embedding_dir)
+            self.initial_embedding_dir = None
```

Comment:
why set the `initial_embedding_dir` to `None` after load the embedding ? If `initial_embedding_dir` is a pretrained emb path there is not matter to set it to `None`

Reply:
To prevent loading an embedding checkpoint repeatedly, when user enter a same `TrainCtx` multiple times.

Reply:
`is_load_pretrained` flag is better than set it to `None`

Reply:
Solved in https://github.com/PersiaML/PersiaML/pull/18/commits/32c28ac5d99b14ea14d0687f27c8031e77422478